src/_pytest/runner.py

```diff
@@ -226,7 +226,7 @@ def call_runtest_hook(item, when: "Literal['setup', 'call', 'teardown']", **kwds
 class CallInfo:
     """ Result/Exception info a function invocation.
 
-    :param excinfo: (`ExceptionInfo`, optional) describing the captured exception if any
+    :param excinfo: (`ExceptionInfo`, optional) describing the captured exception if any.
     :param start: (`float`) The system time when the call started, in seconds since the epoch.
     :param stop: (`float`) The system time when the call ended, in seconds since the epoch.
     :param duration: (`float`) The call duration, in seconds.
```

Comment:
The format is `:param type name:`, e.g. `:param float start:`.
You could also add it below the attribs:
```
start = attr.ib()
"""The system time when the call started, in seconds since the epoch.
:type: float"""
```
Check with `make -C doc/en` though, of course.

Reply:
Thanks for the feedback. I'll simply update the class docstring then, if it does not make much difference. 

Note that I cant check the doc generation as `tox -e docs` does not seem to work on windows 

```
docs run-test: commands[0] | sh -c 'towncrier --draft > doc/en/_changelog_towncrier_draft.rst'
ERROR: InvocationError for command could not find executable sh
```

And with `make` I seem to get an  error 

```
(tools_py37) C:\_dev\python_ws\_Libs_OpenSource\pytest>make -C doc/en html
make: Entering directory `C:/_dev/python_ws/_Libs_OpenSource/pytest/doc/en'
Running Sphinx v2.4.4

Configuration error:
There is a programmable error in your configuration file:

Traceback (most recent call last):
  File "c:\miniconda3\envs\tools_py37\lib\site-packages\sphinx\config.py", line 348, in eval_config_file
    execfile_(filename, namespace)
  File "c:\miniconda3\envs\tools_py37\lib\site-packages\sphinx\util\pycompat.py", line 81, in execfile_
    exec(code, _globals)
  File "C:\_dev\python_ws\_Libs_OpenSource\pytest\doc\en\conf.py", line 22, in <module>
    from _pytest.compat import TYPE_CHECKING
ImportError: cannot import name 'TYPE_CHECKING' from '_pytest.compat' (c:\miniconda3\envs\tools_py37\lib\site-packages\_pytest\compat.py)

make: *** [html] Error 2
make: Leaving directory `C:/_dev/python_ws/_Libs_OpenSource/pytest/doc/en'
```

So for the sake of completing the PR I chose the easy way as you seem to say that any way works ok :)

Reply:
I'm also on Windows, and I use `tox -e docs` to check/generate documentation. 😁

Reply:
@nicoddemus which installer do you use for the missing `sh` required by `tox -e docs` ? (see error message above)
 git bash, cygwin, or some conda package ?

Reply:
Ahh sorry missed that one. My `bash` comes from git:

```
λ where bash
C:\Program Files\Git\usr\bin\bash.exe
```

It is a bit of annoying to require bash on windows just to generate the docs...